packages/zoltan2/core/src/problems/Zoltan2_PartitioningProblem.hpp

```diff
@@ -642,7 +656,10 @@ void PartitioningProblem<Adapter>::solve(bool updateInputData)
   }
   //if mapping is 0 -- coordinate mapping
 
-#if ZOLTAN2_TASKMAPPING_MOVE
+
+//  mapping_type = 0;
+
+#ifdef ZOLTAN2_TASKMAPPING_MOVE_B
```

Comment:
Need comments for this macro; what's happening here?

Reply:
I adjusted this section. I removed the unused ifdef variable and just commented the unused code. Added unblocked portions necessary for the AlgZoltan hier algorithm.